script.js

```diff
@@ -1,10 +1,5 @@
-// function closeMenu() {
-//   document.getElementById("menu").style.display = "none";
-// }
-
 const menubar = document.querySelector('.hamburger');
 const navmenu = document.querySelector('.nav-menu');
-// const open = document.querySelectorAll('.button');
 menubar.addEventListener('click', () => {
   menubar.classList.toggle('active');
   navmenu.classList.toggle('active');
```

Comment:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)for this project, "`When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.`", please make sure that when a user clicks or taps on any of the mobile menu options, it should disappear from the screen. It is currently not the case, please kindly fix that.